pkg/rpc/convert.go

```diff
@@ -155,6 +158,11 @@ func ConvertFromRPCResults(rpcResults []*scanner.Result) []report.Result {
 				}
 			}
 
+			var avdURL string
+			if strings.Contains(strings.ToLower(vuln.VulnerabilityId), "cve") {
+				avdURL = strings.ToLower(fmt.Sprintf("https://avd.aquasec.com/nvd/%s", vuln.VulnerabilityId))
```

Comment:
There are 3 places to generate AVD links. Should we define a function? If we change the URL path, we have to fix 3 places.

Reply:
OK I refactored into a function: https://github.com/aquasecurity/trivy/pull/709/commits/f36239af4b55d256f63bed147b710059f75b3aa0